.github/workflows/ci.yml

```diff
@@ -183,6 +182,8 @@ jobs:
         with:
           reference: benchmark/data/ci-benchmarkData.json
           compare-with: benchmark/build/outputs/connected_android_test_additional_output/releaseAndroidTest/connected/test(AVD) - 10/com.hcaptcha.sdk.bench.test-benchmarkData.json
+      - name: Log diff benchmark reuslts
+        run: echo "${{ steps.diff-benchmark.outputs.markdown-table }}"
```

Comment:
Task `diff-benchmark` does not run on main. It has this conditional:

```
if: github.ref != 'refs/heads/main'
```

Makes sense to remove that conditional as well. @CAMOBAP